### PR TITLE
#5 multienv settings

### PR DESCRIPTION
**Pull request by alxbxbx**
closes #6

### CODE REVIEW

File: pristine/settings/dev.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from pristine.settings.base import *
+
+# Database
+# Still to be defined
```
Comment (kunicmarko20) [inline pristine/settings/dev.py]: newline?

File: pristine/settings/local.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from pristine.settings.base import *
+
+# Database
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+    }
+}
```
Comment (kunicmarko20) [inline pristine/settings/local.py]: newline

File: pristine/settings/prod.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+from pristine.settings.base import *
+
+# Database
+# Still to be defined
```
Comment (kunicmarko20) [inline pristine/settings/prod.py]: newline ...

File: pristine/settings/local.py (hunk 2 above)
Comment (kunicmarko20) [reply pristine/settings/local.py]: What is the difference between dev and local?

File: pristine/settings/dev.py (hunk 1 above)
Comment (alxbxbx) [reply pristine/settings/dev.py]: Mhm, after import part there is one empty line then the code goes.

Comment (kunicmarko20) [reply pristine/settings/dev.py]: @alxbxbx newline missing at the EOF

File: pristine/settings/local.py (hunk 2 above)
Comment (alxbxbx) [reply pristine/settings/local.py]: Maybe we will have development server.

Comment (kunicmarko20) [reply pristine/settings/local.py]: @alxbxbx maybe, never discussed. Can stay for now then.

**Pull request merged**